po/de.po

```diff
@@ -2997,33 +3033,38 @@ msgid ""
 "optimization).\n"
 "    "
 msgstr ""
-"* Das `format!()`-Makro ist eine bequeme Möglichkeit, einen eigenen String aus dynamischen Werten "
-"zu generieren. Es\n"
-"  akzeptiert die gleiche Formatspezifikation wie `println!()`.\n"
+"* `&str` führt einen String-Slice ein, der eine unveränderliche Referenz auf UTF-8-codierte String-"
+"Daten ist, die in einem Speicherblock gespeichert sind. String-Literale (`\"Hallo\"`) werden in "
+"der Binärdatei des Programms gespeichert.\n"
+"\n"
+"* Der `String`-Typ von Rust ist eine Verpackung (wrapper) um einen Byte-Vektor. Wie bei einem "
+"`Vec<T>` ist es im Besitz.\n"
```

Comment:
The sentence "Wie bei einem Vec ist es im Besitz." sounds strange to me, it should state what it owns.

Reply:
Tbh, I also need some clarification here (from the english language). Is it ment that the scope owns the `Vec` or that the `Vec` owns the entries?

Reply:
> Tbh, I also need some clarification here (from the english language). Is it ment that the scope owns the `Vec` or that the `Vec` owns the entries?

Sorry about that, we should clean up the English text to make it clearer!

Here, the difference that the text tries to highlight is that a `&str` value is _borrowed_. When you have `let x: &str = "foo bar"`, the `x` variable does not own the string data. As a consequence, the string data is not deallocated when `x` goes out of scope.

In contrast, a `String` value is _owned_. When you do `let x: String = String::from("foo bar")`, the `x` variable owns the `String`, which internally owns a `Vec<u8>`, which owns every `u8` inside it. So transitively, `x` owns the string data and when `x` goes out of scope, the string data is deallocated.

We can probably make https://github.com/google/comprehensive-rust/blob/main/src/basic-syntax/string-slices.md clearer about this :smile: